### DISCUSSION TITLE
Update for Pi zero 2 w

### DISCUSSION THREAD

**Discussion by Mollusk26** [awawa-dev/HyperHDR#147] [Q&A] [answered]
Could you please update /dependencies/external/rpi_ws281x @5d0041e to the latest version rpi_ws281x @55d7924 

This will add the revisions for the new pi zero w 2 to work as I keep getting
Error hardware revision not supported when setting up HyperHDR.

**Comment by awawa-dev** [comment, marked as answer]
Done

**Reply by Mollusk26** [reply]
Thank you! Hopefully that solves my issue
